person.rb

```diff
@@ -1,15 +1,16 @@
-require './nameable'
+require_relative 'nameable'
 
 class Person < Nameable
-  attr_accessor :id, :name
+  attr_reader :rentals, :name
```

Comment:
Kindly ensure getters and setters are added for name, rentals, and age for the same reasons I stated above😉.